test/test_nestedtensor.py

```diff
@@ -616,17 +616,32 @@ def test_nested_tensor_mul_in_place(self, device, dtype):
     def test_nested_tensor_sum_dim(self, device, dtype):
         params = ((2, (1, 1)), ((4), (4, 4)), (10, (3, 5, 7)))
 
-        def test_sum(nt, dim, keepdim=True):
+        def test_sum(device, dtype, ntensors, max_sizes, dim, keepdim=True, requires_grad=False):
+            nt = self.random_nt(device, dtype, ntensors, max_sizes)
             nt2 = nt.clone()
-            nt = nt.sum(dim=dim, keepdim=keepdim)
             ub2 = nt2.unbind()
-            ub2 = [t.sum(-1, keepdim=keepdim) for t in ub2]
-            nt2 = torch.nested_tensor(ub2)
-            self.assertEqual(nt, nt2)
+            if requires_grad:
+                nt.requires_grad_(requires_grad)
```

Comment:
Why isn't `requires_grad` a kwarg for the `torch.nested_tensor()` constructor? Is that constructor supposed to be differentiable?

Reply:
I believe it is supposed to be differentiable, will look into adding this!

Reply:
If it is differentiable (which looks like it is), you shouldn't add that.